extensions/amp-date-countdown/0.1/amp-date-countdown.js

```diff
@@ -209,15 +216,17 @@ export class AmpDateCountdown extends AMP.BaseElement {
 
     if (this.endDate_) {
       epoch = Date.parse(this.endDate_);
+    } else if (this.timeleftMs_) {
+      epoch = Date.parse(new Date()) + this.timeleftMs_;
     } else if (this.timestampMs_) {
       epoch = this.timestampMs_;
     } else if (this.timestampSeconds_) {
       epoch = this.timestampSeconds_ * 1000;
     }
 
     user().assert(!isNaN(epoch),
-        'One of end-date, timestamp-ms, timestamp-seconds is required');
-
+        'One of end-date, timeleft-ms, timestamp-ms, timestamp-seconds' +
+        'is required');
```

Comment:
nit: you're missing a space between `timestamp-seconds` and `is required`

Reply:
done~